WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -98,11 +100,12 @@
     private ViewGroup mContainer;
     private TextView mTitleTextView;
     private TextView mByLineTextView;
+    private LinearLayout mVersionLayout;
```

Comment:
I don't think we need a property for this, since we can just add the `onClickListener` in `setupViews` with `findViewById(R.id.plugin_version_layout).setOnClickListener()`. It doesn't look like the property is used elsewhere.

Reply:
I replaced the variable in dbbd018.